### PR TITLE
Rarible (Independent Publisher)

### PR DESCRIPTION
**Pull request by RoyPaarMS**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [X] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [X] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
image

- [X] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
image

- [X] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Amjed-Ayoub** [comment]
Hi Roy,
Thank you for all your contributions with us. We appreciate your hard work and time.

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

File: independent-publisher-connectors/Rarible/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,3342 @@
...
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Erc20DecimalBalance"
+            },
+            "headers": {}
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Rarible/apiDefinition.swagger.json:62]: @RoyPaarMS Empty property values in the swagger can be removed until unless you have a specific reason to keep them.

File: independent-publisher-connectors/Rarible/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,3342 @@
...
+    }
+  },
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {},
+  "security": [],
+  "tags": [
+    {
+      "name": "erc20-balance-controller",
+      "description": ""
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Rarible/apiDefinition.swagger.json:3305]: @RoyPaarMS Empty property values can be removed until unless you have a specific reason to keep them.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/Rarible/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,3342 @@
...
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/erc20/balances/{contract}/{owner}": {
+      "get": {
+        "description": "Return ERC20 token balance",
+        "summary": "getErc20Balance",
```
[the remark this answers is not in the document]
Comment (RoyPaarMS) [reply independent-publisher-connectors/Rarible/apiDefinition.swagger.json]: Fixed

File: independent-publisher-connectors/Rarible/apiDefinition.swagger.json (hunk 2 above)
Comment (RoyPaarMS) [reply independent-publisher-connectors/Rarible/apiDefinition.swagger.json]: Removed

File: independent-publisher-connectors/Rarible/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,3240 @@
...
+              "$ref": "#/definitions/Erc20IndexerApiError"
+            }
+          }
+        }
+      }
+    },
+    "/nft/ownerships/{ownershipId}": {
+      "get": {
+        "description": "Return NFTs by Ownership ID",
+        "summary": "getNftOwnershipById",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Rarible/apiDefinition.swagger.json:81]: @RoyPaarMS All the summary and description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid summary and description values. **This comment applies for all the summary values in the swagger.**

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/Rarible/apiDefinition.swagger.json (hunk 4 above)
Comment (RoyPaarMS) [reply independent-publisher-connectors/Rarible/apiDefinition.swagger.json]: Corrected

**Comment by sriyen-msft** [comment]
@RoyPaarMS We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps. Thanks.

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@RoyPaarMS Apologies for the delay in response. 

Congratulations! The deployment of your connector and [documentation page](https://docs.microsoft.com/en-us/connectors/raribleip/) is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**